src/rootcheck/check_rc_dev.c

```diff
@@ -36,9 +36,9 @@ static int read_dev_file(const char *file_name)
     }
 
     else if (S_ISREG(statbuf.st_mode)) {
-        char op_msg[OS_SIZE_1024 + 1];
+        char op_msg[size_name + strlen("File '' present on /dev. Possible hidden file.")  + 1];
 
-        snprintf(op_msg, OS_SIZE_1024, "File '%s' present on /dev."
+        snprintf(op_msg, sizeof(op_msg), "File '%s' present on /dev."
```

Comment:
What's your opinion with the next approach:
- Keep op_msg original size
- Create a variable to store format (with a quantifier) to avoid duplicated ` const char op_msg_fmt[] = "File '%*s' present on /dev. Possible hidden file."`
- Test snprintf required space `const int op_msg_current_size = snprintf(NULL, 0, op_msg_fmt, strlen(file_name), file_name);"`
- If `op_msg_current_size > sizeof(op_msg) - 1`, truncate the `file_name` by using the quantifier with the difference- If not, use strlen as quantifier.